endpoints/globus/endpoint.go

```diff
@@ -317,51 +318,35 @@ func (ep *Endpoint) Transfers() ([]uuid.UUID, error) {
 	return taskIds, nil
 }
 
-func (ep *Endpoint) Transfer(dst core.Endpoint, files []core.FileTransfer) (uuid.UUID, error) {
-	// first, we check that all requested files are staged on this endpoint
-	// (Globus does not perform this check by itself)
-	requestedFiles := make([]core.DataResource, len(files))
-	for i, file := range files {
-		requestedFiles[i].Path = file.SourcePath // only the Path field is required
-	}
-	staged, err := ep.FilesStaged(requestedFiles)
-	if err != nil {
-		return uuid.UUID{}, err
-	}
-	if !staged {
-		return uuid.UUID{}, fmt.Errorf("The files requested for transfer are not yet staged.")
-	}
-
-	// obtain a submission ID
-	// https://docs.globus.org/api/transfer/task_submit/#get_submission_id
-	var xferId uuid.UUID
+// https://docs.globus.org/api/transfer/task_submit/#get_submission_id
+func (ep *Endpoint) getSubmissionId() (uuid.UUID, error) {
+	var id uuid.UUID
```

Comment:
any reason for the variable name change? `xferId` would be consistent with the next function

Reply:
Sorry for the confusion--these functions are different. `getSubmissionId()` returns an ID for submitting a Globus transfer task, after which a transfer task ID is issued.

Reply:
Ah, got it. Wasn't sure if submission ID == transfer ID or not.